client/my-sites/domains/domain-management/list/list-header.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard } from '@automattic/components';
+import InfoPopover from 'components/info-popover';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ListHeader extends React.PureComponent {
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<CompactCard className="list-header">
+				<div className="list__domain-link" />
+				<div className="list__domain-transfer-lock">
+					{ translate( 'Transfer lock' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Transfer lock: off' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-domains-header)